### PR TITLE
Update dotnet templates

### PR DESCRIPTION
**Pull request by cra0zy**
Stuff done:
- Added `.CSharp` to C# templates
- Fixed showing of Content.mgcb from IDEs

### CODE REVIEW

File: ProjectTemplates/DotNetTemplate/MonoGame.Templates.CSharp/content/MonoGame.Application.DesktopGL.CSharp/MGNamespace.csproj [hunk 1]
```diff
@@ -6,7 +6,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <MonoGameContentReference Include="**\*.mgcb" />
+    <MonoGameContentReference Include="Content\Content.mgcb" Visible="false" />
```
Comment (Jjagg) [inline ProjectTemplates/DotNetTemplate/MonoGame.Templates.CSharp/content/MonoGame.Application.DesktopGL.CSharp/MGNamespace.csproj]: Missing none include?

Comment (cra0zy) [reply ProjectTemplates/DotNetTemplate/MonoGame.Templates.CSharp/content/MonoGame.Application.DesktopGL.CSharp/MGNamespace.csproj]: Nope, everything gets autoincluded with either `Compile` or `None` in new csproj format.

Comment (Jjagg) [reply ProjectTemplates/DotNetTemplate/MonoGame.Templates.CSharp/content/MonoGame.Application.DesktopGL.CSharp/MGNamespace.csproj]: Ah, right!

**Comment by Jjagg** [comment]
Thanks @cra0zy! :)

**Pull request merged**